### ISSUE TITLE
Improve @include output.

### ISSUE THREAD

**Issue by silvandiepen** [MatiseAms/henris#12]
At the moment when you want to add multiple output rules. You have to use double brackets:

```css
@include output(('typography','typography-margins')){
// the css
}
```

It would be better if its possible to write this in a cleaner way:

```css
@include output(typography typography-margins){
// the css
}
```

We can also improve this by assuming a dashed argument has a 'parent' argument without the dash (can also be checked if it exists). And automatically also add the 'parent' as an argument event when not provided. 

next will check `$output-typography` too for output. 
```css
@include output(typography-margins){
// the css
}
```

**What we need to do in the output function:**

- add `$arguments...` to enable multiple arguments at once as a list. Go through the list. 
- Add a check for a first dash `-` and check if the variable (parent) exists in the outputs. 
- Add an argument to disable the automatic parent check behaviour by adding a 'false' argument at the end. This can be done with a comma separated argument.

```css
@function output($arguments..., $autoParent: true){
// all output checks. 
}
to use:

@include output(typography-margins){
// the css
// function will check $output-typography-margins & $output-typography
}
or

@include output(typography-margins,false){
// the css
// function will check ONLY $output-typography-margins
}
```

**Comment by silvandiepen** [comment]
@mmeester any thoughts on this?

**Comment by silvandiepen** [comment]
fixed in 0.3.0
